docs/_docs/04_reference/cli-options.md

```diff
@@ -1522,3 +1523,25 @@ plan. Resource actions are indicated with the following symbols:
 
 OpenTofu will perform the following actions:
 ```
+
+### feature
+
+**CLI Arg**: `--feature`<br/>
+**Environment Variable**: `TG_FLAG`<br/>
```

Comment:
Make sure to change the name of the flag in the docs if you agree that we should use `TERRAGRUNT_FLAG` instead.